### DISCUSSION TITLE
[FEATURE REQ] Max download per hour based on GB filter

### DISCUSSION THREAD

**Discussion by rwtallant13** [autobrr/autobrr#1670] [Ideas]
**Is your feature request related to a problem? Please describe.**

Its frustrating not knowing how much storage space you could end up taking up with each hours number of allowed downloads

**Describe the solution you'd like**

Currently the max downloads are based on number of grabs. It would be nice to be able to set that limit so you could set it to grab as many available until 50GB worth every hour for instance.

**Describe alternatives you've considered**

I currently have a min and max size set in the filters and a set number per hour but this still leaves a large amount of variance in how many GB ill be downloading per hour.

**Additional context**

**Comment by KyleSanderson** [comment]
Totally valid FQ. In the meantime, you can use the Size macro to accomplish this as well.
